### ISSUE TITLE
Configure PR github pages deployments to go to separate environment (or sub pages in same environment)

### ISSUE THREAD

**Issue by Polymeta** [3Squared/ForgeUI#43]
Right now, when a PR publishes github pages, they all go to the same github pages environment, which is the main/only environment that forge.ui has at the moment. Right now it might not be an issue, but I imagine this to get confusing when more PRs get opened that people want to check over via the github pages, but then get the deployment of another PR.

**Comment by TomSmith27** [comment]
Yep agreed, unfortunately I'm not aware of any way to do this with github pages

**Comment by Polymeta** [comment]
Yeah, not sure either, but maybe if it allows multiple concurrent deployments, we can sort of make different base URLs, like we did in other places like githubpages/ForgeUI/pullrequestid/ as base url, but not sure if that works

**Comment by TomSmith27** [comment]
Ye looks like this might be coming in the future of github pages, the alternative is to use something like vercel or firebase as they support this out the box, so would be easy

**Comment by Polymeta** [comment]
Yeah vercel looks good, maybe worth setting it up and see how it goes. Comes with bot and CI/CD integration, just needs someone with admin access to set it up
